releases/v20.2.0-beta.1.md

```diff
@@ -0,0 +1,333 @@
+---
+title: What&#39;s New in v20.2.0-beta.1
+toc: true
+summary: Additions and changes in CockroachDB version v20.2.0-beta.1 since version v20.2.0-alpha.3
+---
+
+## August 31, 2020
+
+In addition to various updates, enhancements, and bug fixes, this first v20.2 beta release includes the following major highlights:
+
+- **Backup schedules:** You can now schedule backups natively in CockroachDB. Previously, it was necessary to create your own scheduler service or cron job to kick off [`BACKUP`](../v20.2/backup.html) jobs. You can now set up a recurring schedule that lets CockroachDB handle scheduling, conflict resolution, and resilience for your backups. To create a schedule for a cluster-level backup that runs a full backup every day with the first full backup taken "now", run `CREATE SCHEDULE test_backup_schedule FOR BACKUP INTO ‘<your-backup-storage-location’ RECURRING ‘@daily’ FULL BACKUP ALWAYS WITH EXPERIMENTAL SCHEDULE OPTIONS first_run=NOW`.
```

Comment:
Should we restructure this as:
You can now set up a recurring schedule that lets CockroachDB handle scheduling, conflict resolution, and resilience for your backups. Previously, it was necessary to create your own scheduler service or cron job to kick off [`BACKUP`](../v20.2/backup.html) jobs. To create a schedule for a cluster-level backup that runs a full backup every day with the first full backup taken "now", run `CREATE SCHEDULE test_backup_schedule FOR BACKUP INTO ‘<your-backup-storage-location’ RECURRING ‘@daily’ FULL BACKUP ALWAYS WITH EXPERIMENTAL SCHEDULE OPTIONS first_run=NOW`.